src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -573,10 +573,10 @@ public static LogisticRegressionBinaryTrainer LogisticRegression(this BinaryClas
         }
 
         /// <summary>
-        /// Predict a target using a linear regression model trained with the <see cref="Microsoft.ML.Trainers.PoissonRegressionTrainer"/> trainer.
+        /// Predict a target using a linear regression model trained with the <see cref="Microsoft.ML.Trainers.PoissonRegressionTrainer"/> and advanced options.
```

Comment:
>and [](start = 139, length = 3)

specifying

Reply:
same as before

---
In reply to: [268038068](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268038068) [](ancestors = 268038068)